python/cuspatial/cuspatial/core/binpreds/feature_crosses.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+import cupy as cp
+
+import cudf
+
+from cuspatial.core._column.geocolumn import ColumnType
+from cuspatial.core.binpreds.binpred_interface import NotImplementedRoot
+from cuspatial.core.binpreds.feature_equals import RootEquals
+
+
+class RootCrosses(RootEquals):
+    """Base class for binary predicates that are defined in terms of a
+    the equals binary predicate. For example, a Point-Point Crosses
+    predicate is defined in terms of a Point-Point Equals predicate.
+    """
+
+    def _preprocess(self, lhs, rhs):
+        return self._compute_predicate(lhs, rhs, rhs.point_indices)
+
+    def _compute_predicate(self, lhs, rhs, point_indices):
+        result = super()._compute_predicate(lhs, rhs, point_indices)
+        return self._postprocess(lhs, rhs, point_indices, result)
+
+    def _postprocess(self, lhs, rhs, point_indices, point_result):
+        return super()._postprocess(lhs, rhs, point_indices, point_result)
```

Comment:
Is it necessary to override this if the implementation just calls the parent impl? Or is this a stub to be filled out later?

Reply:
Yeah this is just a stub, in fact I probably just overlooked cleaning it up.